docs/docusaurus/versioned_docs/version-1.8.0/lte/extended_5g_sa_features.md

```diff
@@ -7,7 +7,7 @@ original_id: extended_5g_sa_features
 
 # Extended Features of 5G SA FWA
 
-This document gives an overview of the Extended features added on top of the [integrated 5G SA FWA](https://docs.magmacore.org/docs/lte/integrated_5g_sa) (5G Standalone Architecture Fixed Wireless Access) such as
+This document gives an overview of the Extended features added on top of the [integrated 5G SA FWA](https://magma.github.io/magma/docs/next/lte/integrated_5G_sa) (5G Standalone Architecture Fixed Wireless Access) such as
```

Comment:
Link gives 404 for me. Should be `5g_sa` with small `g` for me.

Reply:
Yes, I fixed this today on master. Probably missed it during rebase.